gradle/libs.versions.toml

```diff
@@ -1,25 +1,25 @@
 [versions]
-android-gradle-plugin = "8.9.2"
+android-gradle-plugin = "8.10.0"
 activity-compose = "1.10.1"
 constraintlayout-compose = "1.1.1"
 compose-material = "1.7.8"
-compose = "2025.04.00"
+compose = "2025.05.00"
 core-ktx = "1.16.0"
 coroutines = "1.10.2"
 coroutines-adapter = "0.9.2"
 dagger-hilt = "2.56.2"
-datastore = "1.1.4"
+datastore = "1.1.6"
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/kseito/RewardedTodo/blob/50f493ab1421e4fbba96bc4ef10078e5d9a0205d/gradle/libs.versions.toml#L11"></span>A newer version of androidx.datastore:datastore-preferences than 1.1.6 is available: 1.1.7</td>
    </tr>
  </tbody>
</table>